### PR TITLE
pixie: remove

### PR DESCRIPTION
**Pull request by armeenm**
###### Description of changes
Pixie [hasn't seen a commit since October 2017](https://github.com/pixie-lang/pixie) and depends on python2.

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.05 Release Notes (or backporting 21.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2205-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by Mindavi** [comment]
cc @bendlas. Also, can you rebase?

**Review by bendlas — APPROVED** [review]
I agree it's not really useful to have in nixpkgs like this ...

**Review by ivan — APPROVED** [review]
going by its GitHub issues, pixie does not seem like it has people trying to use it

**Comment by bendlas** [comment]
cc @alekcz

File: pkgs/top-level/aliases.nix [hunk 1]
```diff
@@ -924,6 +925,7 @@ mapAliases ({
   pgadmin3 = throw "pgadmin3 was removed for being unmaintained, use pgadmin4 instead."; # Added 2022-03-30
   pgp-tools = throw "'pgp-tools' has been renamed to/replaced by 'signing-party'"; # Converted to throw 2022-02-22
   pg_tmp = throw "'pg_tmp' has been renamed to/replaced by 'ephemeralpg'"; # Converted to throw 2022-02-22
+  pixie = throw "pixie has been removed: abandoned by upstream"; # Added 2022-04-21
```
Comment (bendlas) [inline pkgs/top-level/aliases.nix:928]: Would be nice to amend this message with how to install it instead; possibly pointing to NUR.

Comment (Mindavi) [reply pkgs/top-level/aliases.nix:928]: Are you willing to make a follow-up PR for that?

Comment (ivan) [reply pkgs/top-level/aliases.nix:928]: I don't see pixie in nur-combined, so pointing people to NUR wouldn't seem helpful?

Comment (Mindavi) [reply pkgs/top-level/aliases.nix:928]: I'll merge whenever this is rebased. Folloe up can be done if and when it's added to NUR or wherever :)

**Pull request merged**